common/culture/cultures/draenei.txt

```diff
@@ -1,9 +1,10 @@
 ﻿draenei_group = { 
 	graphical_cultures = {
-		western_coa_gfx
-		western_building_gfx
+		polish_coa_gfx
```

Comment:
In vanilla, `polish_coa_gfx` used together with `western_coa_gfx`.

Reply:
Is western_coa_gfx necessary here? The idea was to use polish_coa_gfx alone to give the draenei more uniqueness. Also because western_coa_gfx has a lot of animals and I personally don't seem draenei using a lot of animals due to their arcane and then light obsessions.

Reply:
> Is western_coa_gfx necessary here?

No.